### DISCUSSION TITLE
trying to fix lag on newer versions of JS

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#754] [Ideas]
I'll try closing idle Windows when I'm not really using them that much(need to work on)
changing the garbage frames option to 0
(add more to the comments)

**Comment by NAEL2XD** [comment]
i have some ways to increase performance for js engine:

1. close a shit ton of apps for more memory
2. try this script, might get rid of the lag like a lot but not all of them:
https://github.com/NAEL2XD/naels-psych-lua-scripts/blob/main/Main%20Scripts/Fun%20Scripts/Less%20Lag%20for%20JS%20Engine/script.lua
3. upgrade your cpu/memory
